articles/azure-functions/functions-infrastructure-as-code.md

```diff
@@ -385,129 +524,153 @@ For a sample Azure Resource Manager template, see [Function app on Azure App Ser
 
 ### Create an App Service plan
 
-An App Service plan is defined by a "serverfarm" resource.
+An App Service plan is defined by a "serverfarm" resource. You can specify the SKU as following:
+
+# [Windows](#tab/Windows)
 
 ```json
 {
-    "type": "Microsoft.Web/serverfarms",
-    "apiVersion": "2018-02-01",
-    "name": "[variables('hostingPlanName')]",
-    "location": "[resourceGroup().location]",
-    "sku": {
-        "name": "S1",
-        "tier": "Standard",
-        "size": "S1",
-        "family": "S",
-        "capacity": 1
-    }
+  "type": "Microsoft.Web/serverfarms",
+  "apiVersion": "2021-02-01",
+  "name": "[variables('hostingPlanName')]",
+  "location": "[parameters('location')]",
+  "sku": {
+    "tier": "Standard",
+    "name": "S1",
+    "size": "S1",
+    "family": "S",
+    "capacity": 1
+  }
 }
 ```
 
-To run your app on Linux, you must also set the `kind` to `Linux`:
+# [Linux](#tab/Linux)
+
+To run your app on Linux, you must also set property `"reserved": true` for the serverfarms resource:
 
 ```json
 {
-    "type": "Microsoft.Web/serverfarms",
-    "apiVersion": "2018-02-01",
-    "name": "[variables('hostingPlanName')]",
-    "location": "[resourceGroup().location]",
-    "kind": "Linux",
-    "sku": {
-        "name": "S1",
-        "tier": "Standard",
-        "size": "S1",
-        "family": "S",
-        "capacity": 1
-    }
+  "type": "Microsoft.Web/serverfarms",
+  "apiVersion": "2021-02-01",
+  "name": "[variables('hostingPlanName')]",
+  "location": "[parameters('location')]",
+  "sku": {
+    "tier": "Standard",
+    "name": "S1",
+    "size": "S1",
+    "family": "S",
+    "capacity": 1
+  },
+  "properties": {
+    "reserved": true
+  }
 }
 ```
 
 ### Create a function app
 
-A function app on an App Service plan must have the `serverFarmId` property set to the resource ID of the plan created earlier.
+For function app on a Dedicated plan, you will need to set the `serverFarmId` property on the app so that it points to the resource ID of the plan. You should ensure that the function app has a `dependsOn` setting for the plan as well.
```

Comment:
```suggestion
For function app on a Dedicated plan, you must set the `serverFarmId` property on the app so that it points to the resource ID of the plan. Make sure that the function app has a `dependsOn` setting that also references the plan.
```